src/main/plug/sampler_kernel.cpp

```diff
@@ -26,6 +26,9 @@
 #include <lsp-plug.in/dsp-units/misc/fade.h>
 #include <lsp-plug.in/dsp/dsp.h>
 
+#include <iostream>
```

Comment:
- This is extra unnecessary code.
- Standard C++ headers are strictly prohibited.